### PR TITLE
Group permissions

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: pulpcore/cli/core/context.py [hunk 1]
```diff
@@ -136,6 +136,54 @@ def find(self, **kwargs: Any) -> Any:
...
+        pulp_ctx.needs_plugin("pulpcore", min_version="3.10.dev0")
+        super().__init__(pulp_ctx)
+        self.group_ctx = group_ctx
+
+    def find(self, **kwargs: Any) -> Any:
+        """Workaroud for the missing ability to filter"""
+        # # TODO file a bug and fix upstream
+        # if self.pulp_ctx.has_plugin("pulpcore", min_version="3.10.dev0"):
+        #     # Workaround not needed anymore
+        #     return super().find(**kwargs)
```
Comment (daviddavis) [inline pulpcore/cli/core/context.py:154]: Can this code be removed?

File: CHANGES/100.feature [hunk 2]
```diff
@@ -0,0 +1 @@
+Added label commands.
```
Comment (daviddavis) [inline CHANGES/100.feature:1]: :thinking:

**Comment by daviddavis** [comment]
The code here looks good to me but I am not familiar with rbac in Pulp. May want to get a second opinion from someone who is or I am happy to approve it without that.

File: pulpcore/cli/core/context.py (hunk 1 above)
Comment (mdellweg) [reply pulpcore/cli/core/context.py:154]: No, actually we need figure, whether we can skip the workaround in, say, 3.11.

File: CHANGES/100.feature (hunk 2 above)
Comment (mdellweg) [reply CHANGES/100.feature:1]: :exploding_head:

Comment (mdellweg) [reply CHANGES/100.feature:1]: We don't check that a changelog is only added when there is also a `#fixes` stanza, right?

Comment (daviddavis) [reply CHANGES/100.feature:1]: Yep, the [wrong] assumption is that there wouldn't be a changelog if there is no `fixes #...`

**Review by daviddavis — APPROVED** [review]
LGTM. One suggestion I might make is to add some help text to the `pulp group` command.

**Comment by mdellweg** [comment]
> LGTM. One suggestion I might make is to add some help text to the `pulp group` command.

Not sure what kind of information to add there, but i added help texts to the list, add, and remove commands for group permissions, as the generic ones didn't seem to fit too well.

**Comment by daviddavis** [comment]
I'd say anything with the word "permission" in it to disambiguate "group". So maybe like "manage group permissions"?

**Review by daviddavis — APPROVED** [review]

**Pull request merged**